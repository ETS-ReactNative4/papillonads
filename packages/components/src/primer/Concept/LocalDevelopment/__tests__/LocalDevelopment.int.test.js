import {
  renderSpecificationsStandardsGuidelines,
  renderOpenSourceDependencyProjects,
  renderDesignSystemCore,
  renderProductMicrosites,
} from './LocalDevelopment.int.render'

describe('<LocalDevelopment />', () => {
  describe('Render', () => {
    test('must match renderSpecificationsStandardsGuidelines()', () => {
      expect(global.renderToJSON(renderSpecificationsStandardsGuidelines())).toMatchSnapshot()
    })
    test('must match renderOpenSourceDependencyProjects()', () => {
      expect(global.renderToJSON(renderOpenSourceDependencyProjects())).toMatchSnapshot()
    })
    test('must match renderDesignSystemCore()', () => {
      expect(global.renderToJSON(renderDesignSystemCore())).toMatchSnapshot()
    })
    test('must match renderProductMicrosites()', () => {
      expect(global.renderToJSON(renderProductMicrosites())).toMatchSnapshot()
    })
  })
})
