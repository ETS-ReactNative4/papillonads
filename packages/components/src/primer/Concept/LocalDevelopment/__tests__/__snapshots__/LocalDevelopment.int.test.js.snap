// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LocalDevelopment /> Render must match designSystemCore() 1`] = `
<div>
  <div>
    <h2>
      📗 Papillon Ads Implementation
    </h2>
    <h3>
      📖. Design System Core
    </h3>
    <h4>
      📃. Setup file(s)
    </h4>
    <div>
      ﹘ Coming soon.
    </div>
    <h4>
      📃. Knowledge base
    </h4>
    <div>
      ﹘ Coming soon.
    </div>
  </div>
</div>
`;

exports[`<LocalDevelopment /> Render must match openSourceDependencyProjects() 1`] = `
<div>
  <div>
    <h2>
      📗 Open Source Dependency Projects
    </h2>
    <br />
    <br />
    <h3>
      📖. Webpack DevServer
    </h3>
    <br />
    <div>
      ﹘ 
      <a
        href="https://github.com/webpack/webpack-dev-server"
      >
        Webpack DevServer in Webpack
      </a>
       Serves a webpack app. Updates the browser on changes
    </div>
    <div>
      ﹘ 
      <a
        href="https://webpack.js.org/configuration/dev-server/"
      >
         Webpack DevServer in Webpack Docs
      </a>
       webpack-dev-server can be used to quickly develop an application. See the development guide to get started
    </div>
  </div>
</div>
`;

exports[`<LocalDevelopment /> Render must match productMicrosites() 1`] = `
<div>
  <div>
    <h2>
      📗 Papillon Ads Implementation
    </h2>
    <h3>
      📖. Product Microsites
    </h3>
    <h4>
      📃. Setup file(s)
    </h4>
    <div>
      ﹘ Coming soon.
    </div>
    <h4>
      📃. Knowledge base
    </h4>
    <div>
      ﹘ Coming soon.
    </div>
  </div>
</div>
`;

exports[`<LocalDevelopment /> Render must match specificationsStandardsGuidelines() 1`] = `
<div>
  <div>
    <h2>
      📗 Specifications/Standards/Guidelines
    </h2>
    <div>
      📝. Coming soon 🐬
    </div>
  </div>
</div>
`;
